### PR TITLE
add docs for usage of useccache buildflag

### PR DESCRIPTION
**Pull request by sjamgade**

### CODE REVIEW

File: xml/obs_build_config.xml [hunk 1]
```diff
@@ -187,6 +187,14 @@ Keep: debbuild</screen>
                        will turn to excluded state.</para>
        </listitem>
       </varlistentry>
+      <varlistentry>
+       <term><parameter>useccache:</parameter></term>
+       <listitem>
+           <para>Configure usage of ccache for build of packages mentioned by VALUE. Ccache package will be automatically
+               installed and configured. /.ccache/ dir will be configured as cache dir.
```
Comment (tomschr) [inline xml/obs_build_config.xml:194]: Small addition:

```suggestion
               installed and configured. The directory <filename>/.ccache/</filename> will be configured as cache dir.
```

File: xml/obs_build_config.xml [hunk 2]
```diff
@@ -187,6 +187,14 @@ Keep: debbuild</screen>
...
+               To configure ccache, /.ccache/ccache.conf can be modified as part build process by $BUILD_USER. </para>
```
Comment (tomschr) [inline xml/obs_build_config.xml:195]:
```suggestion
               To configure ccache, the file <filename>/.ccache/ccache.conf</filename> can be modified as part build process by the <envar>$BUILD_USER</envar> environment variable. </para>
```

**Comment by sknorr** [comment]
Thanks! I squashed the two minor corrections into the original commit.

**Comment by sjamgade** [comment]
Thanks for finishing it

**Pull request merged**